tests/test_api_jwt.py

```diff
@@ -419,6 +419,12 @@ def test_check_issuer_when_valid(self, jwt):
         token = jwt.encode(payload, "secret")
         jwt.decode(token, "secret", issuer=issuer, algorithms=["HS256"])
 
+    def test_check_multiple_issuers(self, jwt):
+        issuer = "urn:foo"
+        payload = {"some": "payload", "iss": "urn:foo"}
+        token = jwt.encode(payload, "secret")
+        jwt.decode(token, "secret", issuer=[issuer, "urn:bar"], algorithms=["HS256"])
```

Comment:
The spec insists on issuers being HTTPS URLs:
https://openid.net/specs/openid-connect-core-1_0.html#Terminology

```suggestion
        jwt.decode(token, "secret", issuer=[issuer, "https://other.test"], algorithms=["HS256"])
```

Reply:
(arguably `pyjwt` could be used in other contexts, not just oidc)